Description: Fixes default connection in glance-registry.conf & glance-api.conf
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2014-04-15

Index: glance/etc/glance-api.conf
===================================================================
--- glance.orig/etc/glance-api.conf	2014-09-30 23:18:48.000000000 +0800
+++ glance/etc/glance-api.conf	2014-09-30 23:18:56.000000000 +0800
@@ -584,7 +584,7 @@
 # Deprecated group/name - [DEFAULT]/sql_connection
 # Deprecated group/name - [DATABASE]/sql_connection
 # Deprecated group/name - [sql]/connection
-#connection = <None>
+connection = sqlite:////var/lib/glance/glancedb
 
 # The SQL mode to be used for MySQL sessions. This option,
 # including the default, overrides any server-set SQL mode. To
Index: glance/etc/glance-registry.conf
===================================================================
--- glance.orig/etc/glance-registry.conf	2014-09-30 23:18:48.000000000 +0800
+++ glance/etc/glance-registry.conf	2014-09-30 23:18:56.000000000 +0800
@@ -140,7 +140,7 @@
 # Deprecated group/name - [DEFAULT]/sql_connection
 # Deprecated group/name - [DATABASE]/sql_connection
 # Deprecated group/name - [sql]/connection
-#connection = <None>
+connection = sqlite:////var/lib/glance/glancedb
 
 # The SQL mode to be used for MySQL sessions. This option,
 # including the default, overrides any server-set SQL mode. To
